source/Outfit.cpp

```diff
@@ -43,12 +43,36 @@ const vector<string> Outfit::CATEGORIES = {
 
 void Outfit::Load(const DataNode &node)
 {
+	// Note: an outfit with no name wouldn't be global and therefore
+	// overriding wouldn't make sense (?).
+	bool isOverride = (node.Size() >= 3 && node.Token(2) == "override");
+	if(!name.empty())
+	{
+		if(isOverride)
+		{
+			// If explicitly requested, start from a blank slate,
+			// so that all data is replaced.
+			*this = Outfit();
+		}
+		else
+		{
+			node.PrintTrace("Duplicate definition of outfit "
```

Comment:
I don't agree that we should remove the current behavior of allowing plugins to extend the current definition of an Outfit, e.g. add a single attribute to the existing definition. With this PR, one would then need to fully reproduce the original definition, and additionally include the desired alteration.